### PR TITLE
Partial migration of identity page

### PR DESCRIPTION
**Pull request by filippoweb3**
Removed parts already present in the support documents, added some details, reorg where needed.

### CODE REVIEW

**Comment by filippoweb3** [comment]
Run prettify on the branch, no changes.

File: docs/learn/learn-identity.md [hunk 1]
```diff
@@ -69,8 +69,8 @@ attestation:
 
 - Unknown: The default value, no judgement made yet.
 - Reasonable: The data appears reasonable, but no in-depth checks (e.g. formal KYC process) were
-  performed.
-- Known Good: The registrar has certified that the information is correct.
+  performed (all the currently verified identities).
+- Known Good: The registrar has certified that the information is correct (this step involves passport checks, and at the moment no account has known good identity, with the exception of registrars).
```
Comment (profradha) [inline docs/learn/learn-identity.md]:
```suggestion
- Known Good: The registrar has certified that the information is correct (this step involves verification of state issued identity documents, and at the moment no account has known good identity, with the exception of registrars).
```

File: docs/learn/learn-identity.md [hunk 2]
```diff
@@ -69,8 +69,8 @@ attestation:
 
 - Unknown: The default value, no judgement made yet.
 - Reasonable: The data appears reasonable, but no in-depth checks (e.g. formal KYC process) were
-  performed.
-- Known Good: The registrar has certified that the information is correct.
+  performed (all the currently verified identities).
```
Comment (profradha) [inline docs/learn/learn-identity.md]:
```suggestion
  performed (all the currently verified identities on-chain).
```

File: docs/learn/learn-identity.md [hunk 3]
```diff
@@ -97,6 +97,11 @@ The procedure to request and cancel identity judgements is explained in detail i
 
 :::
 
+:::info Requesting judgement through Web3 Foundation Registrar
+If you requested judgement for your on-chain identity through the Web3 Foundation Registrar (i.e. Registrar #0) you will need to complete a few additional challenges. For more information visit [this support article](https://support.polkadot.network/support/solutions/articles/65000179747-how-to-use-the-w3f-registrar-page).
```
Comment (profradha) [inline docs/learn/learn-identity.md]:
```suggestion
If you requested judgement for your on-chain identity through the Web3 Foundation Registrar (i.e. Registrar #0) you will need to complete a few additional tasks. For more information visit [this support article](https://support.polkadot.network/support/solutions/articles/65000179747-how-to-use-the-w3f-registrar-page).
```

**Review by DrW3RK — APPROVED** [review]
Looks good!

**Pull request merged**